hack/lib/init.sh

```diff
@@ -156,3 +156,28 @@ kube::realpath() {
   fi
   kube::readlinkdashf "$1"
 }
+
+function kube::build::ensure_docker_daemon_connectivity {
```

Comment:
please put this in `hack/lib/util.sh` instead; `hack/lib/init.sh` loads the other scripts in `hack/lib`, but shouldn't contain any other functionality itself.

also s/build/util/ in the name, to match the rest of the functions in `hack/lib/util.sh`.